docs/api/package.json

```diff
@@ -34,82 +34,14 @@
       "type": "image/png"
     }
   ],
-  "assets": [
```

Comment:
What about this missing content here?

Reply:
Fixed. I didn't know I have to load assets additionally.